consensus/types/src/eth_spec.rs

```diff
@@ -146,6 +146,11 @@ pub trait EthSpec:
     /// Must be set to `BytesPerFieldElement * FieldElementsPerCell`.
     type BytesPerCell: Unsigned + Clone + Sync + Send + Debug + PartialEq;
 
+    /// The maximum number of cell commitments per block
+    ///
+    /// MaxBlobCommitmentsPerBlock * FieldElementsPerExtendedBlob / FieldElementsPerCell
+    type MaxCellCommitmentsPerBlock: Unsigned + Clone + Sync + Send + Debug + PartialEq;
```

Comment:
Should rename this to `MaxCellsPerBlock`, there's no such thing as cell commitments. (commitments are for blobs)

Reply:
Addressed in https://github.com/sigp/lighthouse/pull/7117/commits/a5f9649c74aa79abbac90057b0e039c0bc0053eb